src/main/java/art/heredium/domain/coupon/entity/Coupon.java

```diff
@@ -61,6 +61,14 @@ public class Coupon extends BaseEntity {
   @Column(name = "image_url")
   private String imageUrl;
 
+  @Comment("사용횟수")
+  @Column(name = "number_of_uses", nullable = true)
+  private Long numberOfUses;
+
+  @Comment("상시할인")
+  @Column(name = "permanent")
+  private Boolean permanent;
```

Comment:
```suggestion
  @Column(name = "is_permanent")
  private Boolean isPermanent;
```
Để thống nhất với isEnabled.

Reply:
Đã fix cùng cfb2f77bb460f11a8550ec9d898a25fb2a1228db